### PR TITLE
Do not create the kairos user by default

### PR DESCRIPTION
**Pull request by Itxaka**
Part of https://github.com/kairos-io/kairos/issues/2921

### CODE REVIEW

**Comment by Itxaka** [comment]
Tested kairos with this, doesnt seem like an issue. You cannot login but thats about it lol.

If you create a normal user you can still login but without sudo privs unless you add the admin group to your user

```
$ sudo -i
[sudo] password for itxaka: 
itxaka is not in the sudoers file.
$ cat /et	pas^C
$ bash
itxaka@localhost:~$ cat /etc/passwd
root:x:0:0:root:/root:/bin/bash
daemon:x:1:1:daemon:/usr/sbin:/usr/sbin/nologin
bin:x:2:2:bin:/bin:/usr/sbin/nologin
sys:x:3:3:sys:/dev:/usr/sbin/nologin
sync:x:4:65534:sync:/bin:/bin/sync
games:x:5:60:games:/usr/games:/usr/sbin/nologin
man:x:6:12:man:/var/cache/man:/usr/sbin/nologin
lp:x:7:7:lp:/var/spool/lpd:/usr/sbin/nologin
mail:x:8:8:mail:/var/mail:/usr/sbin/nologin
news:x:9:9:news:/var/spool/news:/usr/sbin/nologin
uucp:x:10:10:uucp:/var/spool/uucp:/usr/sbin/nologin
proxy:x:13:13:proxy:/bin:/usr/sbin/nologin
www-data:x:33:33:www-data:/var/www:/usr/sbin/nologin
backup:x:34:34:backup:/var/backups:/usr/sbin/nologin
list:x:38:38:Mailing List Manager:/var/list:/usr/sbin/nologin
irc:x:39:39:ircd:/run/ircd:/usr/sbin/nologin
_apt:x:42:65534::/nonexistent:/usr/sbin/nologin
nobody:x:65534:65534:nobody:/nonexistent:/usr/sbin/nologin
ubuntu:x:1000:1000:Ubuntu:/home/ubuntu:/bin/bash
messagebus:x:100:101::/nonexistent:/usr/sbin/nologin
_rpc:x:101:65534::/run/rpcbind:/usr/sbin/nologin
systemd-network:x:998:998:systemd Network Management:/:/usr/sbin/nologin
statd:x:102:65534::/var/lib/nfs:/usr/sbin/nologin
systemd-timesync:x:997:997:systemd Time Synchronization:/:/usr/sbin/nologin
tss:x:103:103:TPM software stack,,,:/var/lib/tpm:/bin/false
sshd:x:104:65534::/run/sshd:/usr/sbin/nologin
Debian-snmp:x:105:104::/var/lib/snmp:/bin/false
_lldpd:x:106:105::/run/lldpd:/usr/sbin/nologin
polkitd:x:992:992:User for polkitd:/:/usr/sbin/nologin
systemd-resolve:x:991:991:systemd Resolver:/:/usr/sbin/nologin
itxaka:x:1001:1001:Created by entities:/home/itxaka:/bin/sh
```

permissions of the user seems good
```
itxaka@localhost:~$ k3s kubectl get nodes
error: error loading config file "/etc/rancher/k3s/k3s.yaml": open /etc/rancher/k3s/k3s.yaml: permission denied
itxaka@localhost:~$ cat /oem/90_custom.yaml
cat: /oem/90_custom.yaml: Permission denied
```

So I dont see why this would not work, nothing depends on our kairos user being there by default.

File: packages/static/kairos-overlay-files/files/system/oem/09_openrc_services.yaml [hunk 1]
```diff
@@ -111,7 +111,6 @@ stages:
           ╚═╝  ╚═╝╚═╝  ╚═╝╚═╝╚═╝  ╚═╝ ╚═════╝ ╚══════╝
                                                       
 
-          Welcome to kairos (login with user: kairos, password: kairos)
```
Comment (mauromorales) [inline packages/static/kairos-overlay-files/files/system/oem/09_openrc_services.yaml:114]: maybe still worth keeping for livecd? though I don't see why someone would log out of the livecd

**Review by mauromorales — APPROVED** [review]

**Review by jimmykarily — APPROVED** [review]

File: packages/static/kairos-overlay-files/files/system/oem/09_openrc_services.yaml (hunk 1 above)
Comment (Itxaka) [reply packages/static/kairos-overlay-files/files/system/oem/09_openrc_services.yaml:114]: I think you can't log out of the livecd lol, it restarts the service on exit so you get logged in back

Let me try that

Comment (Itxaka) [reply packages/static/kairos-overlay-files/files/system/oem/09_openrc_services.yaml:114]: nope, I think this only happens in alpine, which makes the service restart but not on normal service. You dont even get a login back, just a dead tty lol

**Pull request merged**